packages/client/src/views/OfficeHome/sentForReview/SentForReview.tsx

```diff
@@ -216,9 +218,11 @@ class SentForReviewComponent extends React.Component<
           )) ||
         ''
       const sentForApproval =
-        (reg.createdAt && Number.isNaN(Number(reg.createdAt))
-          ? new Date(reg.createdAt)
-          : new Date(Number(reg.createdAt))) || ''
+        getPreviousOperationDateByOperationType(
+          reg.operationHistories,
+          this.isFieldAgent ? RegStatus.Declared : RegStatus.Validated
+        ) || ''
+
```

Comment:
@euanmillar For declarations that don't have `operationHistories`, should we show some fallback value(which would be wrong) or not show anything at all?